engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -113,11 +121,11 @@ def announce_system_scc(auth, params)
       JSON.parse(response.body)
     end
 
-    def scc_activate_product(product, auth, token, email)
+    def scc_activate_product(product, auth, params, mode)
```

Comment:
please explain the change in method parameters

Reply:
Before, the mode was `byos` now we need to send something to indicate `hybrid` as well, 
so instead of 3 parameter (`token`, `email` and `mode`), I decided to send 2 (`params` which include `token` and `email` and `mode`)

Reply:
I think it would have been more straightforward, and require less change, to just add the `mode` argument to the function. I don't see the value in also changing the behavior of  `token` and `email` arguments.